sdk/servicebus/azure-servicebus/tests/async_tests/test_sessions_async.py

```diff
@@ -1170,4 +1170,29 @@ async def test_async_session_non_session_send_to_session_queue_should_fail(self,
             async with sb_client.get_queue_sender(servicebus_queue.name) as sender:
                 with pytest.raises(ServiceBusError):
                     message = ServiceBusMessage("Handler message")
-                    await sender.send_messages(message)
\ No newline at end of file
+                    await sender.send_messages(message)
+
+    @pytest.mark.asyncio
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', requires_session=True)
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasserAsync()
+    async def test_async_next_available_session_timeout_value(self, uamqp_transport, *, servicebus_namespace_connection_string=None, servicebus_queue=None, **kwargs):
+        if uamqp_transport==True:
+            pytest.skip("This test is for pyamqp only")
+        async with ServiceBusClient.from_connection_string(
+            servicebus_namespace_connection_string, logging_enable=False, uamqp_transport=uamqp_transport) as sb_client:
+            
+            receiver = sb_client.get_queue_receiver(servicebus_queue.name, session_id=NEXT_AVAILABLE_SESSION, max_wait_time=10)
```

Comment:
for the sake of thoroughness, would we want to do the same thing right below in the same test with max_wait_time>65?